apps/prairielearn/src/components/QuestionContainer.html.ts

```diff
@@ -655,19 +659,45 @@ function QuestionPanel({
             questionNumber: instance_question_info?.question_number,
           })}
         </h1>
-        ${showCopyQuestionButton
-          ? html`
-              <button
-                class="btn btn-light btn-sm ms-auto"
-                type="button"
-                data-bs-toggle="modal"
-                data-bs-target="#copyQuestionModal"
-              >
-                <i class="fa fa-clone"></i>
-                Copy question
-              </button>
-            `
-          : ''}
+        <div class="ms-auto d-flex flex-row gap-1">
+          <div class="btn-group">
+            ${showCopyQuestionButton
+              ? html`
+                  <button
+                    class="btn btn-sm btn-outline-light"
+                    type="button"
+                    data-bs-toggle="modal tooltip"
+                    data-bs-title="Copy question"
+                    data-bs-target="#copyQuestionModal"
+                  >
+                    <i class="fa fa-fw fa-clone"></i>
+                  </button>
+                `
+              : ''}
+            ${manualGradingPreviewUrl
+              ? html`
+                  <div class="btn-group">
```

Comment:
If I'm understanding this correctly, it looks like this `btn-group` is nested inside the top-level `btn-group`?

Reply:
TIL! I guess I've done this before and just forgot, thanks for the link!